api/collections.py

```diff
@@ -328,6 +328,7 @@ def get(self, cid):
         for sess in sessions:
             sess['measurements'] = session_measurements.get(sess['_id'], None)
             sess['subject_code'] = sess.get('subject', {}).get('code', '') # FIXME when subject is pulled out of session
+            sess['attachment_count'] = len([f for f in sess.get('files', []) if f.get('flavor') == 'attachment'])
```

Comment:
What is this counter used for? Mainly I'm wondering which user interface features need this specific count, since we could probably count a lot of different attributes...

For example, I think you could get this client-side with a simple `_.filter(array, function(x) { return x.flavor == "attachment" }).length` if you're using underscore.